src/transformStateWithClones.js

```diff
@@ -8,6 +9,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const table = {
+    ...state,
+  };
+  const result = [];
+
+  for (const action of actions) {
+    if (action.type === 'clear') {
```

Comment:
[CODE STYLE] - Instead of using a `for...in` loop and deleting properties one by one, you can simply set `table` to an empty object to clear it. This is more efficient and clearer in intent.